change/@azure-msal-node-2020-11-09-16-30-14-sni.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Should this be prerelease?
```suggestion
  "type": "prerelease",
```

---

Done